util/httputil/context.go

```diff
@@ -21,9 +21,11 @@ import (
 	"github.com/prometheus/prometheus/promql"
 )
 
-type ctxParam int
+type ctxParam string
 
-var pathParam ctxParam
+const (
+	pathParam ctxParam = "path_param"
+)
```

Comment:
```suggestion
const pathParam ctxParam = "path"
```

Reply:
:+1:

Reply:
I'd suggest moving to a one liner with shorter value.

Reply:
updated.